go.mod

```diff
@@ -4,7 +4,7 @@ go 1.19
 
 require (
 	github.com/CosmWasm/cosmwasm-go v0.5.1-0.20220822092235-974247a04ac7
-	github.com/CosmWasm/wasmd v0.29.2
+	github.com/CosmWasm/wasmd v0.32.0
```

Comment:
we're on a fork and since we're using replace this was not synced, so I synced it just for visibility sake.